### ISSUE TITLE
Searched videos buffer with both options enabled

### ISSUE THREAD

**Issue by Kuddus73** [YT-Advanced/YT-Advanced#20]
I'm on v2.187.3 with both spoofing options enabled.
Videos on the homepage and subscriptions page play fine, but any searched video gets stuck after a minute and something:

https://github.com/YT-Advanced/YT-Advanced/assets/127908708/9eac4d34-f3ba-412d-8cbd-a41579db15a8

**Comment by Adity1243** [comment]
Yes, most of videos get stop at 1:00 to 1:15 middle of these timeline, when video plays if it comes after 1:05 its starts buffering and get black screen in video and never plays we have restart that video but same problem happen continuos with both spoof player parameter enabled .

**Comment by UntaDotMy** [comment]
i fix the buffer with spoof parameter type enable . no buffer so far 
![Screenshot_20230813_032842_YouTube](https://github.com/YT-Advanced/YT-Advanced/assets/42847931/6d2cebbd-7a53-487a-8f0d-156cbb76e3b3)

**Comment by Kuddus73** [comment]
> i fix the buffer with spoof parameter type enable . no buffer so far !

Did you even read the description of the issue?

**Comment by Kuddus73** [comment]
> i fix the buffer with spoof parameter type enable . no buffer so far !

Did you even read the description of the issue or at least the title?

**Comment by YT-Advanced** [comment]
Cannot reproduce it on my device 🤔

**Comment by Kuddus73** [comment]
These problems with buffering started from v2.187.3, when you added the option "Spoof player parameter type".
I went back to v2.187.2 which doesn't have that option and it has no buffering issues.

**Comment by UntaDotMy** [comment]
> > i fix the buffer with spoof parameter type enable . no buffer so far !
> 
> Did you even read the description of the issue or at least the title?

i understand the title and the description , just mention that i enable the spoof parameter type and the buffering issue on my device is so far no buffer , if i turn off this spoof , it start buffering again , tested with normal video and searched video .  tested with latest test patch from ReX and 2.187.3 .

**Comment by YT-Advanced** [comment]
> These problems with buffering started from v2.187.3, when you added the option "Spoof player parameter type".
I went back to v2.187.2 which doesn't have that option and it has no buffering issues.

When you turn on spoof player parameter type, did the seekbar thumbnail show in search video ?

**Comment by Kuddus73** [comment]
Please watch the video fully:

https://github.com/YT-Advanced/YT-Advanced/assets/127908708/7c786c08-fa44-43e0-b1e6-2a70d64f19b8

**Comment by YT-Advanced** [comment]
Hmm, I will try to fix it, not sure I can do it or not because I can't reproduce it :(

**Comment by Kuddus73** [comment]
I got both v2.187.2 and v2.187.3 installed by having different package names.
Is it possible to install the test v2.190.0 through Revancify?
I wanna see if that version has the same issue.

**Comment by YT-Advanced** [comment]
> I got both v2.187.2 and v2.187.3 installed by having different package names. Is it possible to install the test v2.190.0 through Revancify? I wanna see if that version has the same issue.

Now is nope. The test version will continue get this issues, I think

**Comment by YT-Advanced** [comment]
> I got both v2.187.2 and v2.187.3 installed by having different package names. Is it possible to install the test v2.190.0 through Revancify? I wanna see if that version has the same issue.

Can you perform a clean install. It may have you solve the problem

**Comment by Kuddus73** [comment]
I did a clean install twice.
Can I ask you why you added the option "Spoof player parameter type"?

Cuz I have both v2.187.2 and v2.187.3 installed right now and in the first version no beffering issues.

**Comment by Adity1243** [comment]
I am using inotia00 last revanced extended patch and it has no buffering issues i tested.

**Comment by Adity1243** [comment]
> Please watch the video fully:
> 
>  Screen_Recording_20230813_114632_YouTube.mp4

I can see that you played home screen video and ambient mode is not working in that video but when you searched the video and played it the ambient mode is working that mean either patch is broken in searched videos or coded only for home screen and subscription videos.

**Comment by Adity1243** [comment]
Also did you enabled new thumbnail seekbar option ?

> Please watch the video fully:
> 
>  Screen_Recording_20230813_114632_YouTube.mp4

**Comment by YT-Advanced** [comment]
> I did a clean install twice. Can I ask you why you added the option "Spoof player parameter type"?

Some user mention me that Short parameter worked on their device

> I can see that you played home screen video and ambient mode is not working in that video but when you searched the video and played it the ambient mode is working that mean either patch is broken in searched videos or coded only for home screen and subscription videos.

Oh, I see. I think I will have a fix for this case right now

**Comment by Adity1243** [comment]
Glad i can help.

**Comment by YT-Advanced** [comment]
@Kuddus73 Check back in 5 minutes and repatch. If it worked please notify me !

**Comment by Kuddus73** [comment]
@YT-Advanced Yep, that fixed, thank you.
